### PR TITLE
introduce pallet utility and re-run benchmark 

### LINKED ISSUE #72: include utility pallet
utility is needed to process batch extrinsics, which we want to use

### PR DESCRIPTION
**Pull request by brenzi**
fixes #72 
benchmarked on devsgx02

### CODE REVIEW

File: runtime/src/weights/frame_system.rs [hunk 1]
```diff
@@ -29,46 +30,46 @@ use sp_std::marker::PhantomData;
 pub struct WeightInfo<T>(PhantomData<T>);
 impl<T: frame_system::Config> frame_system::WeightInfo for WeightInfo<T> {
 	fn remark(b: u32, ) -> Weight {
-		(574_000 as Weight)
+		(0 as Weight)
```
Comment (brenzi) [inline runtime/src/weights/frame_system.rs]: we need to fix this I guess

File: runtime/src/lib.rs [hunk 2]
```diff
@@ -534,6 +534,12 @@ impl pallet_scheduler::Config for Runtime {
 	type WeightInfo = weights::pallet_scheduler::WeightInfo<Runtime>;
 }
 
+impl pallet_utility::Config for Runtime {
+	type Event = Event;
+	type Call = Call;
+	type WeightInfo = ();
```
Comment (clangenb) [inline runtime/src/lib.rs]: Here you need to fill in the actual weight info, just like above on line 534.

You need to declare the `pallet_utility` module manually in the weights module.

File: runtime/src/weights/frame_system.rs [hunk 3]
```diff
@@ -34,41 +35,41 @@ impl<T: frame_system::Config> frame_system::WeightInfo for WeightInfo<T> {
 			.saturating_add((1_000 as Weight).saturating_mul(b as Weight))
 	}
 	fn remark_with_event(b: u32, ) -> Weight {
-		(0 as Weight)
+		(2_653_000 as Weight)
```
Comment (clangenb) [inline runtime/src/weights/frame_system.rs:38]: These are the ones, where I really wonder about the stability...

**Review by clangenb — COMMENTED** [review]
Minor comment, the rest looks good.

I am surprised that the weights are very similar to my laptop.

**Review by clangenb — APPROVED** [review]
Will still approve, so you can merge after the tiny fix.

File: runtime/src/weights/frame_system.rs (hunk 3 above)
Comment (brenzi) [reply runtime/src/weights/frame_system.rs:38]: yes, but I'd rather go with the more pessimistic value here....

**Pull request merged**